src/StatusQ/Controls/StatusInput.qml

```diff
@@ -419,7 +424,8 @@ Item {
             height: visible ? contentHeight : 0
             font.pixelSize: 12
             color: Theme.palette.dangerColor1
-            Layout.alignment: Qt.AlignVCenter
+            Layout.alignment: Qt.AlignVCenter | Qt.AlignTop
```

Comment:
`Qt.AlignVCenter | Qt.AlignTop` for real? :) I'm confused, and QML will be as well :D

Reply:
true! vcenter removed